app/models/news_item.rb

```diff
@@ -43,8 +47,29 @@ def self.chart_data(is_topcoin = false)
       .order_by_published(:asc)
   end
 
-  def coin_link_data
-    coins.map { |coin| coin.as_json(only: [:symbol, :slug, :id] ) }
+  def tag_scoped_coins
+    case ENV['NEWS_COIN_MENTION_TAG_SCOPE']
```

Comment:
We try to use `ENV.fetch('NEWS_COIN_MENTION_TAG_SCOPE')` when the env var is required so that the app throws an error if it's missing.